lib/class-wp-theme-json-gutenberg.php

```diff
@@ -383,9 +383,10 @@ class WP_Theme_JSON_Gutenberg {
 			'minHeight' => null,
 		),
 		'layout'                        => array(
-			'contentSize'  => null,
-			'wideSize'     => null,
-			'allowEditing' => null,
+			'contentSize'            => null,
+			'wideSize'               => null,
+			'allowEditing'           => null,
+			'allowCustomContentSize' => null,
```

Comment:
I wonder if it might be worth adding some of the other settings that allow for disabling controls, such as `allowOrientation` or `allowJustification`. Currently these only work when set in block.json.

Reply:
Great idea! To keep things contained, I reckon we could explore those in separate PRs?